toolkit/themes/shared/design-system/design-tokens.json

```
@@ -135,40 +101,42 @@
         "value": "#5a3100"
       }
     },
-    "white": "#ffffff"
-  },
-  "text": {
-    "color": {
-      "prefers-contrast": {
-        "value": "CanvasText"
-      },
-      "deemphasized": {
-        "prefers-contrast": {
-          "value": "inherit"
-        }
-      },
-      "brand": {
-        "light": {
-          "value": "{color.gray.100}"
-        },
-        "dark": {
-          "value": "{color.gray.05}"
-        },
-        "deemphasized": {
-          "light": {
-            "value": "color-mix(in srgb, currentColor 69%, transparent)"
-          },
-          "dark": {
-            "value": "color-mix(in srgb, currentColor 75%, transparent)"
-          }
-        }
+    "white": {
+      "value": "#ffffff"
+    },
+    "background": {
+      "critical": {
+        "value": "{color.red.05}",
+        "darkValue": "{color.red.80}"
```

```suggestion
        "dark": "{color.red.80}"
```

nit: I thought we were gonna drop the `Value` suffix to make things more concise? I still don't have strong opinions about this but we should probably pick one and stick with it (happy to update my patch if we want to use it)